web-api/openapi/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.3.0 - 2023-06-27
```

Comment:
:art: we probably should update the date to the current day right before merging